src/lib.rs

```diff
@@ -57,3 +59,6 @@ pub use crate::array_string::ArrayString;
 pub use crate::errors::CapacityError;
 
 pub use crate::arrayvec::{ArrayVec, IntoIter, Drain};
+
+#[cfg(feature="copy")]
+pub use crate::arrayvec_copy::ArrayVecCopy;
```

Comment:
There are other public items inside  `arrayvec_copy` that we need to make reachable for the users - for example its iterator types. It's likely it needs to be in its own module, what do you think?

Reply:
Yes, its own module would make using it easier.